docs/user/supported_devices.rst

```diff
@@ -79,7 +79,7 @@ ar71xx-generic
   - Archer C59 (v1)
   - Archer C7 (v2, v4, v5)
   - CPE210 (v1.0, v1.1, v2.0, v3.0)
-  - CPE220 (v1.1)
+  - CPE220 (v1.1, v3.0)
```

Comment:
Would need to be put into the ath79-generic target, not ar71xx-generic. However I'm believe we need to flag this device as BROKEN, seeing as it uses the ath10k driver and only features 64 MB of RAM. 

CC @ecsv

Reply:
Section fixed